packfile.h

```diff
@@ -86,6 +86,13 @@ extern void unuse_pack(struct pack_window **);
 extern void clear_delta_base_cache(void);
 extern struct packed_git *add_packed_git(const char *path, size_t path_len, int local);
 
+/*
+ * Close the pack, unlink the .pack and associated extension files.
+ * Does not unlink if 'ignore_keep' is true and the pack-file is
+ * marked as ".keep".
+ */
+extern void unlink_pack(const char *pack_name, int ignore_keep);
```

Comment:
Nit: maybe "pack_path" since you're now exposing it publicly and taking a full path name
(with suffix) whereas the old interface took a directory path name and the pack file base
name (without suffix).   A casual reader might assume "pack_name" is the base name.